src/test/java/ScheduleExporterTest.java

```diff
@@ -0,0 +1,37 @@
+import entities.Schedule;
+import java.io.IOException;
+import java.util.ArrayList;
+import org.junit.Before;
+import org.junit.Test;
+import workers.ScheduleExporter;
+import workers.Scheduler;
+
+public class ScheduleExporterTest {
+
+    ScheduleExporter exporter;
+    Schedule inPersonSchedule, onlineSchedule;
+
+    @Before
+    public void setUp() throws IOException {
+        // TODO: if we mock api, get course for mocked course. If not, hardcode in a session
+        ArrayList<String> inPerson = new ArrayList<>();
+        ArrayList<String> online = new ArrayList<>();
+        inPerson.add("MAT237");
+        online.add("STA247");
+        Scheduler scheduler = new Scheduler();
+        inPersonSchedule = scheduler.createBasicSchedule(inPerson);
+        onlineSchedule = scheduler.createBasicSchedule(online);
+
+        exporter = new ScheduleExporter();
+    }
+
+    @Test(timeout = 100)
+    public void testSessionExportInPerson() {
+        // TODO: Fill this in later
```

Comment:
Check [this](https://stackoverflow.com/questions/3840356/how-to-test-in-java-that-a-class-implements-serializable-correctly-not-just-is) link out. Essentially you go:
`new ObjectOutputStream(new ByteArrayOutputStream()).writeObject(myObject);`
and make sure no error is thrown.